MdeModulePkg/Include/Library/DeviceStateLib.h

```diff
@@ -22,6 +22,11 @@ MU_CHANGE: new file
 #define DEVICE_STATE_UNIT_TEST_MODE            (1 << (5))
 #define DEVICE_STATE_MEM_PROTECTIONS_OFF       (1 << (6))
 
+#define DEVICE_STATE_PLATFORM_MODE_4           (1 << (20))
```

Comment:
John - As far as you are aware is any entity using the raw value?  Should we just move 0-3 down in bit field and then add 4-7 on top?  Since this hasn't been upstreamed to edk2 we have more flexibility.

Reply:
AFAIK, no one is using the raw value; I'd be happy to move it around so the bits are in order if you prefer - I just wasn't sure how baked the existing definitions were.

Reply:
Everything we have a macro based and should handle it fine....Transition for a device in the field should be transparent as well since these are not cached but detected on each boot, right?

Reply:
Updated definitions.